src/lib.rs

```diff
@@ -1500,61 +1407,58 @@ impl EGraph {
         Ok(())
     }
 
-    fn process_command(
-        &mut self,
-        command: Command,
-        show_egglog_only: bool,
-    ) -> Result<Vec<ResolvedNCommand>, Error> {
-        let program = desugar::desugar_program(vec![command], &mut self.parser, self.seminaive)?;
+    fn process_command(&mut self, command: Command) -> Result<Vec<ResolvedNCommand>, Error> {
+        let mut program = self.resolve_command(command)?;
 
-        let mut program = self.typecheck_program(&program)?;
-
-        if !show_egglog_only {
-            program = remove_globals::remove_globals(program, &mut self.parser.symbol_gen);
-
-            for command in &program {
-                self.names.check_shadowing(command)?;
-            }
+        program = remove_globals::remove_globals(program, &mut self.parser.symbol_gen);
```

Comment:
Is this change compatible with egglog-experimental's resugar tests?

Reply:
I believe so because it still allows you to run just the desugarring that doesn't run remove globals?